### DISCUSSION TITLE
How to bypass the chat/main app authentication ?

### DISCUSSION THREAD

**Discussion by RockyBabbi** [Azure-Samples/chat-with-your-data-solution-accelerator#1371] [Q&A]
I deployed the code successfully but not able to register the app in Entra as I'm not the tenant admin. Is there a way to bypass the app authentication to bring up the chat app ? admin app works great.

<img width="1069" alt="image" src="https://github.com/user-attachments/assets/4b138307-fee9-4f7e-b237-0b6c1969d5b4">

**Comment by Roopan-Microsoft** [comment]
Hi @RockyBabbi, no we cannot skip this as we have implemented the Chat History and maintaining user specific history. 

You can ask the tenant admin to add as this is one time step.

Let me know if any other help required. Happy to Help!

**Comment by Roopan-Microsoft** [comment]
Hey @RockyBabbi - Just following up if your issue is resolved.

**Comment by RockyBabbi** [comment]
Hi Roopan,Thanks for checking. I’m good for now.Sent from my iPhoneOn Oct 14, 2024, at 10:22 AM, Roopan-Microsoft ***@***.***> wrote:﻿
Hey @RockyBabbi - Just following up if your issue is resolved.

**Reply by Roopan-Microsoft** [reply]
Thats great to hear! :)
